src/Ethereum/ABI/Array.cpp

```diff
@@ -104,6 +104,9 @@ bool ParamArray::setValueJson(const std::string& value) {
 }
 
 Data ParamArray::hashStruct() const {
+    if (empty) {
```

Comment:
This is array related, you can leave out this file.

Reply:
Oh, my bad, missing something